gef-ui/templates/partials/confirm-cover-start-date.njk

```diff
@@ -35,13 +37,56 @@
         </div>
     </div>
 
-    <h1 data-cy="confirm-cover-start-date-heading">Confirm cover start date for issued facilities</h1>
+    {% if facility.length > 0 %}
+        {% if success %}
+            {{
+                mojBanner({
+                type: 'success',
+                text: success.message,
+                iconFallbackText: 'Success'
+                })
+            }}
+        {% endif %}
+    {% endif %}
 
-    {% if isUkefReviewAvailable %}
+    {% if isUkefReviewPositive %}
         <div class="govuk-grid-row govuk-body">
             <div class="govuk-grid-column-two-thirds">
-                <form method="POST" autocomplete="off">
-                </form>
+                {% if facility.length > 0 %}
+                    {{ govukTable
+                        ({
+                                caption: "Confirm cover start date for issued facilities",
+                                captionClasses: "govuk-table__caption--xl",
+                                firstCellIsHeader: true,
+                                classes: "govuk-table__vertical-align",
+                                head: [
+                                    {
+                                    text: "Faciliy name"
+                                    },
+                                    {
+                                    text: "UKEF Facility ID"
+                                    },
+                                    {
+                                    text: "Value"
+                                    },
+                                    {
+                                    }
+                                ],
+                                rows: facility
+                        })
+                    }}
+                {% else %}
+                    {{
+                        mojBanner({
+                        type: 'success',
+                        text: 'All cover start dates confirmed for issued facilities',
+                        iconFallbackText: 'Success'
+                        })
+                    }}
+                    <div class="govuk-button-group">
+                        <a href = '/gef/application-details/{{ applicationId }}' class = 'govuk-button govuk-button--primary'>Continue</a>
```

Comment:
do we need this to be a `govukButton` instead?

Reply:
Done